app/views/home/contributors.md

```diff
@@ -0,0 +1,69 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that might be looking for ways to use those skills on an open source project.
```

Comment:
I think the audience is developers, but to @andrew's vision (which I share), we should emphasize that contributing to OSS doesn't just mean writing code. So if you spend your Fridays working on documentation, or triaging issues, that's great too.

This is also why I don't think the CTA should necessarily be "make a PR".